src/resources/views/crud/fields/repeatable.blade.php

```diff
@@ -56,26 +57,46 @@
           border: 1px solid rgba(0,40,100,.12);
           border-radius: 5px;
           background-color: #f0f3f94f;
+          margin-bottom: 20px !important;
         }
-        .container-repeatable-elements .delete-element {
+        .container-repeatable-elements .action-element {
           z-index: 99;
           position: absolute!important;
-          margin-left: -24px;
-          margin-top: 0px;
           height: 30px;
           width: 30px;
           border-radius: 15px;
           text-align: center;
           background-color: #e8ebf0!important;;
         }
+
+        .container-repeatable-elements .delete-element {
+					top: -15px;
+					right: -15px;
+        }
+
+        .container-repeatable-elements .move-element {
+					top: 15px;
+					left: -15px;
+					outline: none;
+					cursor: move;
+					line-height: 29px;
+        }
```

Comment:
I think rather than having two places for the buttons (left and right), we could have a single place, to the left. One showing the x sign, one showing the arrows in all directions sign, so it's clear it can be dragged. 

That would make for a better interface in my opinion because at a glance you know what you can do to the items - delete or move.

Reply:
You said jQuery UI is already here ? But I dont see it, and have an error on sortable function if I remove the script ! Do we have to include jQuery UI in the config file or something like that ?

Reply:
@tabacitu Okay, I follow your recommandations ! But I think in the future, a toolbar will be good : delete, reorder, clone .. right ?